### PR TITLE
ENG-653: Set up new motif powered docs site

### PR DESCRIPTION
**Pull request by crondinini**
## Description

This PR does 2 things:

- Adds a middleware that exposes the docs from Motif
- Allows image from cloudinary (used by motif)
- Syncs the files from Motif with our repo

### CODE REVIEW

File: middleware.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @next/next/no-server-import-in-page */
```
Comment (crondinini) [inline middleware.tsx]: For some reason eslint complains both if I add `pages/_middleware` AND if I import this here. This is the way it seems to work, but eslint still complains. Let me know if you know how I can fix this.

File: motif-docs/components/Badge.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Badge } from 'https://cdn.motif.land/motifblocks@0.0.4'
```
Comment (crondinini) [inline motif-docs/components/Badge.mdx]: Everything from `motif-docs` is a sync. You can ignore all of it.
To see real changes, check first and last commit only.

File: middleware.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  // These are the paths used in motif
+  const documentationPaths = [
+    '/documentation',
+    '/troubleshooting',
+    '/http-api',
+    '/troubleshooting',
+    '/changelog',
+  ]
```
Comment (flybayer) [inline middleware.tsx:20]: I feel like ALL motif pages should be under `flightcontrol.dev/docs`, that will make it easier for us to reason about what is served from motif vs not.

File: motif-docs/package.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
+{
+  "type": "module",
+  "scripts": {
+    "dev": "next dev -p 4000",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@headlessui/react": "0.0.0-insiders.ab78fbd",
```
Comment (flybayer) [inline motif-docs/package.json:10]: you need to install `sharp` so image transforms are fast

Comment (crondinini) [reply motif-docs/package.json:10]: @flybayer Do I need to do anything other than that?

File: flightcontrol.json [hunk 5]
```diff
@@ -21,8 +21,80 @@
...
+      },
+      "services": [
+        {
+          "id": "website",
+          "name": "Website",
+          "type": "fargate",
+          "buildType": "nixpacks",
+          "cpu": 0.25,
+          "memory": 0.5,
+          "domain": "www.flightcontrol.dev",
```
Comment (flybayer) [inline flightcontrol.json]: not supported. I made a linear issue to not accept this
```suggestion
```

**Pull request merged**